docs/uninstall-olm.md

```diff
@@ -0,0 +1,46 @@
+# How do I uninstall OLM?
+
+OLM is installed within a namespace called `openshift-operator-lifecycle-manager` on an OpenShift cluster. On a local minikube cluster, it is installed within the namespace `olm`. You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl delete ns <OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>
+```
+
+To find out the version of the OLM you are using, please use the following to see the version of packageserver csv.
+
+```bash
+oc -n <OLM namespace, e.g.: olm, openshift-operator-lifecycle-management, etc> get csvs
+NAME DISPLAY VERSION REPLACES PHASE
+packageserver Package Server 0.13.0 Succeeded
+```
+
+## Uninstall From Git Repository Master Branch
+
+You can also uninstall OLM from the master branch of the [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) repository with the following: 
+
+```bash
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl delete ns <OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>
```

Comment:
Instead of  `<OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>`, do you think <namespace-olm-in-installed-in> conveys the message in a more tidy way?

Reply:
Maybe without the  OpenShift specific namespace it will be much better?

Reply:
Sure we could just use the ns `olm` if you want, provided we have some sort of note at the top of the document that says `all the commands below assume that olm in installed in the ns olm`. That way anybody installing olm in a different namespace can just swap out the name.